templates/scitas/arvine.yaml.j2

```diff
@@ -42,7 +42,7 @@
   # - chip-seq
   - eigen
   - fastqc
-  - ffmpeg
```

Comment:
You cannot remove an already installed package, You can only add new ones
This might need a suffix in the modules to differentiate the modules